spec/requests/authentication_test_spec.rb

```diff
@@ -0,0 +1,45 @@
+# require 'rails_helper'
```

Comment:
Why is this commented?

Reply:
I do not know if it's necessary this spec file.. we are testing Devise here.. and it suppose to be tested. I suggest you to delete it, WDYT @jpascual1994?